Flaky `test_profile.py::TestProfile::test_profile_variable `

I have seen this one fail repeatedly recently, such as in https://github.com/pymc-devs/pymc/runs/8135110596?check_suite_focus=true

```python
 ================================== FAILURES ===================================
______________________ TestProfile.test_profile_variable ______________________

self = <pymc.tests.test_profile.TestProfile object at 0x0000020C722FB460>

    def test_profile_variable(self):
        rv = self.model.basic_RVs[0]
>       assert self.model.profile(self.model.logp(vars=[rv], sum=False)).fct_call_time
E       assert 0.0
E        +  where 0.0 = <aesara.compile.profiling.ProfileStats object at 0x0000020C729943D0>.fct_call_time
E        +    where <aesara.compile.profiling.ProfileStats object at 0x0000020C729943D0> = <bound method Model.profile of <pymc.model.Model object at 0x0000020C728B4130>>([Elemwise{mul,no_inplace}.0])
E        +      where <bound method Model.profile of <pymc.model.Model object at 0x0000020C728B4130>> = <pymc.model.Model object at 0x0000020C728B4130>.profile
E        +        where <pymc.model.Model object at 0x0000020C728B4130> = <pymc.tests.test_profile.TestProfile object at 0x0000020C722FB460>.model
E        +      and   [Elemwise{mul,no_inplace}.0] = <bound method Model.logp of <pymc.model.Model object at 0x0000020C728B4130>>(vars=[x], sum=False)
E        +        where <bound method Model.logp of <pymc.model.Model object at 0x0000020C728B4130>> = <pymc.model.Model object at 0x0000020C728B4130>.logp
E        +          where <pymc.model.Model object at 0x0000020C728B4130> = <pymc.tests.test_profile.TestProfile object at 0x0000020C722FB460>.model

pymc\tests\test_profile.py:27: AssertionError
```

--- a/pymc/tests/test_model.py
+++ b/pymc/tests/test_model.py
@@ -1138,11 +1138,11 @@ class TestProfile:
         _, self.model, _ = simple_model()
 
     def test_profile_model(self):
-        assert self.model.profile(self.model.logp()).fct_call_time > 0
+        assert self.model.profile(self.model.logp(), n=3000).fct_call_time > 0
 
     def test_profile_variable(self):
         rv = self.model.basic_RVs[0]
-        assert self.model.profile(self.model.logp(vars=[rv], sum=False)).fct_call_time
+        assert self.model.profile(self.model.logp(vars=[rv], sum=False), n=3000).fct_call_time > 0
 
     def test_profile_count(self):
         count = 1005

